scripts/find_replace.sh

```diff
@@ -41,6 +42,7 @@ if [ "$CIRCLE_BRANCH" == "master" ]; then
     ESTUARY_PINNING_SECRET=$ESTUARY_PINNING_SECRET_MASTER
     WEB3_STORAGE_PINNING_SECRET=$WEB3_STORAGE_PINNING_SECRET_MASTER
     ETHERSCAN_API_KEY=$ETHERSCAN_API_KEY_MASTER
+    CREATE2_CLIENT_TOKENS=$CREATE2_CLIENT_TOKENS_MASTER
 fi
 
 for VAR_NAME in INFURA_ID ALCHEMY_ID AWS_S3_ACCESS_KEY_ID AWS_S3_SECRET_ACCESS_KEY IPFS_SECRET NPM_TOKEN PUBLIC_IP LOCAL_IP SESSION_SECRET ALCHEMY_ID_OPTIMISM ALCHEMY_ID_ARBITRUM CHAINSAFE_S3_ACCESS_KEY_ID CHAINSAFE_S3_SECRET_ACCESS_KEY ESTUARY_PINNING_SECRET WEB3_STORAGE_PINNING_SECRET ETHERSCAN_API_KEY
```

Comment:
Needs to be added to to-be-replaced variables
```suggestion
for VAR_NAME in INFURA_ID ALCHEMY_ID AWS_S3_ACCESS_KEY_ID AWS_S3_SECRET_ACCESS_KEY IPFS_SECRET NPM_TOKEN PUBLIC_IP LOCAL_IP SESSION_SECRET ALCHEMY_ID_OPTIMISM ALCHEMY_ID_ARBITRUM CHAINSAFE_S3_ACCESS_KEY_ID CHAINSAFE_S3_SECRET_ACCESS_KEY ESTUARY_PINNING_SECRET WEB3_STORAGE_PINNING_SECRET ETHERSCAN_API_KEY CREATE2_CLIENT_TOKENS
```